server.js

```diff
@@ -6,12 +6,15 @@ var app = express();
 
 const rootDir = path.join(__dirname, 'build');
 app.use(express.static(rootDir));
-app.get('/*', (req, res) => {
+app.get('/*splat', (req, res) => {
```

Comment:
The new version of express requests (no pun intended) this.

See the full migration guide at https://expressjs.com/en/guide/migrating-5.html